src/rpcrawtransaction.cpp

```diff
@@ -731,3 +731,41 @@ Value sendrawtransaction(const Array& params, bool fHelp)
 
     return hashTx.GetHex();
 }
+
+Value getspentzerocoinamount(const Array& params, bool fHelp)
+{
+    if (fHelp || params.size() != 2)
+        throw runtime_error(
+            "getspentzerocoinamount hexstring index\n"
+            "\nReturns value of spent zerocoin output designated by transaction hash and input index.\n"
+            "\nArguments:\n"
+            "1. hash          (hexstring) Transaction hash\n"
+            "2. index         (int) Input index\n"
+            "\nResult:\n"
+            "\"value\"        (int) Spent output value, -1 if error\n"
+            "\nExamples:\n" +
+            HelpExampleCli("getspentzerocoinamount", "78021ebf92a80dfccef1413067f1222e37535399797cce029bb40ad981131706 0"));
+
+    uint256 txHash = ParseHashV(params[0], "parameter 1");
+    int inputIndex = params[1].get_int();
+    if (inputIndex < 0)
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter for transaction input");
+
+    CTransaction tx;
+    uint256 hashBlock = 0;
+    if (!GetTransaction(txHash, tx, hashBlock, true))
+        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "No information available about transaction");
+
+    if (inputIndex >= (int)tx.vin.size())
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter for transaction input");
+
+    const CTxIn& input = tx.vin[inputIndex];
+    if (!input.scriptSig.IsZerocoinSpend())
+        return -1;
+
+    std::vector<char, zero_after_free_allocator<char> > dataTxIn;
+    dataTxIn.insert(dataTxIn.end(), input.scriptSig.begin() + 4, input.scriptSig.end());
+    CDataStream serializedCoinSpend(dataTxIn, SER_NETWORK, PROTOCOL_VERSION);
+    libzerocoin::CoinSpend spend(Params().Zerocoin_Params(), serializedCoinSpend);
+    return libzerocoin::ZerocoinDenominationToAmount(spend.getDenomination());
```

Comment:
Should return the formatted version of the value. Most people have a hard time figuring out pivtoshis.
```
CAmount nValue = libzerocoin::ZerocoinDenominationToAmount(spend.getDenomination());
return FormatMoney(nValue);
```